tap.d.ts

```diff
@@ -0,0 +1,39 @@
+// tap.d.ts
+
+declare module Test {
+
+  class Deferred {
+    resolve();
+    reject();
+  }
+
+  class test {
```

Comment:
Shouldn't the module be `tap` and then this be `class Test`?  I'm not a TypeScript expert, so maybe I'm just not grokking this :)